test/payload.test.js

```diff
@@ -0,0 +1,99 @@
+const Rapptor = require('rapptor');
+const tap = require('tap');
+const path = require('path');
+const util = require('util');
+
+process.env.ENDPOINT = '/payload';
+const workDir = path.resolve(__dirname, '../');
+
+const wait = util.promisify(setTimeout);
+
+let rapptor;
+const start = async function() {
+  rapptor = new Rapptor({
+    cwd: workDir
+  });
+
+  await rapptor.setup();
+  await rapptor.start();
+
+  return true;
+};
+
+const stop = async function() {
+  await rapptor.stop();
+
+  return true;
+};
+
+
+tap.test('gathers object confgurations', async (t) => {
+  await start();
```

Comment:
I think you should pass in the configuration into this function so we can see what it is without going back to test.yaml